code/controllers/voting.dm

```diff
@@ -1,9 +1,15 @@
 var/datum/controller/vote/vote = new()
 
-var/global/list/round_voters = list() //Keeps track of the individuals voting for a given round, for use in forcedrafting.
+var/global/list/round_voters = list() // Keeps track of the individuals voting for a given round, for use in forcedrafting.
 
-datum/controller/vote
-	var/initiator = null
+#define VOTE_RESTART "restart"
+#define VOTE_GAMEMODE "gamemode"
+#define VOTE_CREW_TRANSFER "crew_transfer"
+#define VOTE_ADD_ANTAGONIST "add_antagonist"
+#define VOTE_CUSTOM "custom"
+

```

Comment:
My point was that add this in comments so that when someone breaks said file X amount of months down the road, and they are looking at the file, they can right off the bat tell what going on.